docs/lib/tasks/helper.rb

```diff
@@ -69,14 +69,14 @@ module Docs
     CONNECTION_ARGS = %w[after before first last].to_set
 
     FIELD_HEADER = <<~MD
-      #### Fields
+      ###### **Fields**
 
```

Comment:
Changes looking good. :rocket:.

Quick side note, also push the generated `md` file in this PR.